vpn-router-web/src/main/java/vpnrouter/web/ClientsView.java

```diff
@@ -0,0 +1,24 @@
+package vpnrouter.web;
+
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.router.PageTitle;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.spring.annotation.UIScope;
+import org.springframework.stereotype.Component;
+
+@UIScope
+@Component
+@PageTitle("Clients")
+@Route("")
+public class ClientsView extends VerticalLayout {
```

Comment:
Why do we need this page?)) The main page should contain the clients table and other control elements

Reply:
Oh, this page is redundant now) I deleted it